### PR TITLE
Add yaml var

### PR DESCRIPTION
**Pull request by CohenIdo**
- add data yaml as var
- fix `elastic package check` errors

### CODE REVIEW

File: packages/cis_kubernetes_benchmark/manifest.yml [hunk 1]
```diff
@@ -30,5 +30,14 @@ policy_templates:
       - type: kubebeat
         title: Enable CIS Kubernetes Benchmark
         description: Collecting findings
+vars:
+  - name: dataYaml
+    type: yaml
+    title: Rules Activation Yaml
+    multi: false
+    required: false
+    show_user: true
```
Comment (eyalkraft) [inline packages/cis_kubernetes_benchmark/manifest.yml:39]: please hide from user

**Comment by kfirpeled** [comment]
please bump version number

**Comment by kfirpeled** [comment]
another issue is when adding integration for the first time the data is missing the proper formatting:
Screen Shot 2022-02-28 at 14 58 07

**Comment by CohenIdo** [comment]
@kfirpeled, I don't see a reason to merge this PR. it can use us for test [update package policy PR](https://github.com/build-security/kibana/pull/133). 
This task [solve](https://github.com/elastic/security-team/issues/3058) is in the right way.

**Review by kfirpeled — CHANGES_REQUESTED** [review]
see comments ^^

**Review by kfirpeled — APPROVED** [review]

**Pull request merged**